x-pack/test/functional/apps/transform/creation_index_pattern.ts

```diff
@@ -308,39 +359,59 @@ export default function ({ getService }: FtrProviderContext) {
             testData.expected.histogramCharts
           );
 
-          await transform.testExecution.logTestStep('adds the group by entries');
-          for (const [index, entry] of testData.groupByEntries.entries()) {
-            await transform.wizard.assertGroupByInputExists();
-            await transform.wizard.assertGroupByInputValue([]);
-            await transform.wizard.addGroupByEntry(
-              index,
-              entry.identifier,
-              entry.label,
-              entry.intervalLabel
+          if (isPivotTransformTestData(testData)) {
+            await transform.testExecution.logTestStep('adds the group by entries');
+            for (const [index, entry] of testData.groupByEntries.entries()) {
+              await transform.wizard.assertGroupByInputExists();
+              await transform.wizard.assertGroupByInputValue([]);
+              await transform.wizard.addGroupByEntry(
+                index,
+                entry.identifier,
+                entry.label,
+                entry.intervalLabel
+              );
+            }
+
+            await transform.testExecution.logTestStep('adds the aggregation entries');
+            await transform.wizard.addAggregationEntries(testData.aggregationEntries);
+
+            await transform.testExecution.logTestStep('displays the advanced pivot editor switch');
+            await transform.wizard.assertAdvancedPivotEditorSwitchExists();
+            await transform.wizard.assertAdvancedPivotEditorSwitchCheckState(false);
+
+            await transform.testExecution.logTestStep('displays the advanced configuration');
+            await transform.wizard.enabledAdvancedPivotEditor();
```

Comment:
I think there's a typo here. Should this be `enableAdvancedPivotEditor`?

Reply:
good spot, it's been here for a while, fixed in https://github.com/elastic/kibana/pull/86966/commits/bcaa7dfb1b3c2c40bbbc210c46749180d898efff